doc/sailfish-webview-webview.qdoc

```diff
@@ -66,6 +68,505 @@
+  //       what if the content is smaller than the viewport size?
+*/
+
+/*!
+  \qmlproperty QPointF WebView::scrollableOffset
+  \readonly
+  \brief The (x and y pixel) offset into the scrollable size which describes
+         the top-left pixel of the content being displayed by the viewport
+
+  The value of this property is effectively equal to the value of the
+  x and y values of the \l{contentRect} after applying the appropriate
+  \l{resolution} scale factor (to map from CSS pixel coordinates to
+  device-resolution pixel coordinates).
+
+  \sa scrollableSize, contentRect, resolution
+*/
+
+/*!
+  \qmlproperty bool WebView::atXBeginning
+  \readonly
+  \brief Whether the viewport of the webview is scrolled to the far-left of
+         the content.
+
+  Effectively, the value of this property will be true if the scrollableOffset
+  x-value is approximately zero.
+
+  \sa atXEnd, scrollableOffset
+*/
+
+/*!
+  \qmlproperty bool WebView::atXEnd
+  \readonly
+  \brief Whether the viewport of the webview is scrolled to the far-right of
+         the content.
+
+  Effectively, the value of this property will be true if the
+  \l{scrollableOffset} x-value plus the (\l{resolution} scaled)
+  \l{contentRect} width is approximately equal to or greater than the
+  \l{scrollableSize} width.
+
+  \sa atXEnd, scrollableOffset, scrollableSize, contentRect
+*/
+
+/*!
+  \qmlproperty bool WebView::atYBeginning
+  \readonly
+  \brief Whether the viewport of the webview is scrolled to the top of
+         the content.
+
+  Effectively, the value of this property will be true if the scrollableOffset
+  y-value is approximately zero.
+
+  \sa atYEnd, scrollableOffset
+*/
+
+/*!
+  \qmlproperty bool WebView::atYEnd
+  \readonly
+  \brief Whether the viewport of the webview is scrolled to the bottom of
+         the content.
+
+  Effectively, the value of this property will be true if the
+  \l{scrollableOffset} y-value plus the (\l{resolution} scaled)
+  \l{contentRect} height is approximately equal to or greater than the
+  \l{scrollableSize} height.
+
+  \sa atYEnd, scrollableOffset, scrollableSize, contentRect, resolution
+*/
+
+/*!
+  \qmlproperty float WebView::resolution
+  \readonly
+  \brief The scaling factor which is applied to the (CSS-pixel) contentRect
+         values to map to the appropriate device-resolution pixel coordinates
+
+  The value of this property is equal to the width of the device's screen
+  divided by the width of the \l{contentRect}.  As such, it will update
+  when the user zooms in or out (thus changing the width, in CSS pixels, of
+  the contentRect), or if the page orientation changes (resulting in a change
+  to landscape orientation from portrait orientation, or vice-versa).
+
+  \sa contentRect
+*/
+
+/*!
+  \qmlproperty bool WebView::painted
+  \readonly
+  \brief Whether the first paint has completed
+*/
+
+/*!
+  \qmlproperty bool WebView::bgcolor
```

Comment:
This should be called "backgroundColor"

Reply:
Created JB#54342 / JOLLA-158

Reply:
not changed yet, will document as-is and then we can make changes later as part of that separate bug I guess